### DISCUSSION TITLE
Ascend/Descend choice for columns?

### DISCUSSION THREAD

**Discussion by jet1671** [LesFerch/WinSetView#78] [Ideas]
I would like to sort by Date Modified, Descending (latest file modified at the top).
If this is possible, let me know how to do it (I can't figure it out).
Or else, it would be a much appreciated feature.
When I choose to sort by Date Modified, it presents as Ascending.
Great tool, thanks so much for making it available!

**Comment by jet1671** [comment]
Oops, solved - the little box at the top of the sort selection column has an arrow "↑" in it. (It looked like a "t"; I didn't understand)
Click on it, it becomes "↓" and my desire to sort descending is fulfilled! 
Thanks -

**Comment by LesFerch** [comment]
👍
